tests/order/UpdateOrderMutation.graphql

```diff
@@ -0,0 +1,12 @@
+mutation ($email: String, $orderId: ID!, $status: String) {
```

Comment:
Does `customFields` need to be added in the mutation here, or is that going to ben an extension in its' own package?

Reply:
It will need to be added if used because we don't know what the schema should be.